### PR TITLE
[#36841] Bump CKEditor to v32

### PR DESCRIPTION
**Pull request by oliverguenther**
[OP#36841](https://community.openproject.com/wp/36841)

- [x] Load UI translations dynamically depending on the user's language.
- [x] Fix custom classes being applied due to changes in ImageBlock and ImageInline (new plugin now disabled by us).

:warning: Merge together with https://github.com/opf/commonmark-ckeditor-build/pull/72

**To check**

- Any new plugins to add without further impacts?

### CODE REVIEW

**Comment by oliverguenther** [comment]
codeql complains about the vendored ckeditor, not much we can do about those.

File: frontend/src/app/shared/components/editor/components/ckeditor/ckeditor-setup.service.ts [hunk 1]
```diff
@@ -99,11 +104,26 @@ export class CKEditorSetupService {
...
+    }
+
+    try {
+      await import(
+        /* webpackChunkName: "ckeditor-translation" */ `../../../../../../vendor/ckeditor/translations/${I18n.locale}.js`
+      ) as unknown;
+      return I18n.locale;
+    } catch (e:unknown) {
+      console.warn(`Failed to load translation for CKEditor: ${e as string}`);
+      return 'en';
```
Comment (b12f) [inline frontend/src/app/shared/components/editor/components/ckeditor/ckeditor-setup.service.ts]: It's a bit weird that an editor `load` function returns a locale name string. I'd probably just set the locale in this method.

Comment (oliverguenther) [reply frontend/src/app/shared/components/editor/components/ckeditor/ckeditor-setup.service.ts]: Yeah good point, I'll change that.

**Review by b12f — COMMENTED** [review]
LGTM

**Pull request merged**